packages/schematics/angular/guard/index_spec.ts

```diff
@@ -93,9 +91,8 @@ describe('Guard Schematic', () => {
 
   it('should respect the implements values', async () => {
     const implementationOptions = ['CanActivate', 'CanLoad', 'CanActivateChild'];
-    const options = { ...defaultOptions, implements: implementationOptions};
-    const tree = await schematicRunner.runSchematicAsync('guard', options, appTree)
-      .toPromise();
+    const options = { ...defaultOptions, implements: implementationOptions };
```

Comment:
Next time please avoid done unnecessary styling changes, as it caused chum and makes reviewing harding. Thanks.

Reply:
yes, I will do that. In this case it was just wrongly formatted. See curly braces.